posthog/hogql/constants.py

```diff
@@ -1,87 +1,145 @@
+    # logical
+    "and": ("and", 2, None),
+    "or": ("or", 2, None),
+    "xor": ("xor", 2, None),
+    "not": ("not", 1, 1),
     # type conversions
-    "toInt": "toInt64OrNull",
-    "toFloat": "toFloat64OrNull",
-    "toDecimal": "toDecimal64OrNull",
-    "toDate": "toDateOrNull",
-    "toDateTime": "toDateTimeOrNull",
-    "toUUID": "toUUIDOrNull",
-    "toString": "toString",
-    # intervals
-    "toIntervalSecond": "toIntervalSecond",
-    "toIntervalMinute": "toIntervalMinute",
-    "toIntervalHour": "toIntervalHour",
-    "toIntervalDay": "toIntervalDay",
-    "toIntervalWeek": "toIntervalWeek",
-    "toIntervalMonth": "toIntervalMonth",
-    "toIntervalQuarter": "toIntervalQuarter",
-    "toIntervalYear": "toIntervalYear",
-    # date functions
-    "now": "now",
-    "NOW": "now",
-    "toMonday": "toMonday",
-    "toStartOfYear": "toStartOfYear",
-    "toStartOfQuarter": "toStartOfQuarter",
-    "toStartOfMonth": "toStartOfMonth",
-    "toStartOfWeek": "toStartOfWeek",
-    "toStartOfDay": "toStartOfDay",
-    "toStartOfHour": "toStartOfHour",
-    "toStartOfMinute": "toStartOfMinute",
-    "toStartOfSecond": "toStartOfSecond",
-    "toStartOfFiveMinutes": "toStartOfFiveMinutes",
-    "toStartOfTenMinutes": "toStartOfTenMinutes",
-    "toStartOfFifteenMinutes": "toStartOfFifteenMinutes",
-    "toTimezone": "toTimezone",
-    "age": "age",
-    "dateDiff": "dateDiff",
-    "dateTrunc": "dateTrunc",
-    "formatDateTime": "formatDateTime",
-    # string functions
-    "length": "lengthUTF8",
-    "empty": "empty",
-    "notEmpty": "notEmpty",
-    "leftPad": "leftPad",
-    "rightPad": "rightPad",
-    "lower": "lower",
-    "upper": "upper",
-    "repeat": "repeat",
-    "format": "format",
-    "concat": "concat",
-    "coalesce": "coalesce",
-    "substring": "substringUTF8",
-    "appendTrailingCharIfAbsent": "appendTrailingCharIfAbsent",
-    "endsWith": "endsWith",
-    "startsWith": "startsWith",
-    "trim": "trimBoth",
-    "trimLeft": "trimLeft",
-    "trimRight": "trimRight",
-    "extractTextFromHTML": "extractTextFromHTML",
-    "match": "match",
-    "like": "like",
-    "ilike": "ilike",
-    "notLike": "notLike",
-    "replace": "replace",
-    "replaceOne": "replaceOne",
-    # array functions
-    "tuple": "tuple",
-    # conditional
-    "if": "if",
-    "not": "not",
-    "multiIf": "multiIf",
-    # rounding
-    "round": "round",
-    "floor": "floor",
-    "ceil": "ceil",
-    "trunc": "trunc",
+    "toInt": ("toInt64OrNull", 1, 1),
+    "toFloat": ("toFloat64OrNull", 1, 1),
+    "toDecimal": ("toDecimal64OrNull", 1, 1),
+    "toDate": ("toDateOrNull", 1, 1),
+    "toDateTime": ("toDateTimeOrNull", 1, 1),
+    "toUUID": ("toUUIDOrNull", 1, 1),
+    "toString": ("toString", 1, 1),
+    "toJSONString": ("toJSONString", 1, 1),
+    "parseDateTime": ("parseDateTimeOrNull", 2, 2),
+    "parseDateTimeBestEffort": ("parseDateTimeBestEffortOrNull", 2, 2),
+    # dates and times TODO
```

Comment:
I also suggest adding `today`, saw a user need it a couple weeks ago (`toDate(now())` just doesn't roll of the tongue as well)